frame/support/procedural/src/pallet/parse/storage.rs

```diff
@@ -480,15 +482,16 @@ fn process_named_generics(
 	let metadata = generics.metadata()?;
 	let query_kind = generics.query_kind();
 
-	Ok((Some(generics), metadata, query_kind))
+	Ok((Some(generics), metadata, query_kind, false))
```

Comment:
So it is not possible to use with named generics?

Reply:
The syntax only works in `dev_mode` where we are trying to simplify the call. Do we need a dev_mode pallet with named generic but `_` hasher?

Reply:
I think its fine. Just wanted to ensure that i understand it :smile: